doc/man7/ossl-guide-quic-introduction.pod

```diff
@@ -0,0 +1,135 @@
+It provides similar security guarantees to TLS such as confidentiality,
+integrity and authentication (see L<ossl-guide-tls-introduction(7)>). It
+additionally provides multiplexing capabilities through the use of "streams"
+(see L</QUIC STREAMS> below).
+
+=head1 QUIC TIME BASED EVENTS
+
+A key difference between the TLS implementation and the QUIC implementation in
+OpenSSL is how time is handled. The QUIC protocol requires various actions to be
+performed on a regular basis regardless of whether application data is being
+transmitted or received.
+
+OpenSSL introduces a new function L<SSL_handle_events(3)> that will
+automatically process any outstanding time based events that must be handled.
+Alternatively calling any I/O function such as L<SSL_read_ex(3)> or
+L<SSL_write_ex(3)> will also process these events. There is also
+L<SSL_get_event_timeout(3)> which tells an application the amount of time that
+remains until L<SSL_handle_events(3)> (or any I/O function) must be called.
+
+Fortunately a blocking application that does not leave the QUIC connection idle,
+and is regularly calling I/O functions does not typically need to worry about
+this. However if you are developing a nonblocking application or one that may
+leave the QUIC connection idle for a period of time then you will need to
+arrange to call these functions.
+
+OpenSSL provides an optional "thread assisted mode" that will automatically
+create a background thread and will regularly call L<SSL_handle_events(3)> in a
+thread safe manner. This provides a simple way for an application to satisfy the
+QUIC requirements for time based events without having to implement special
+logic to accomplish it.
+
+=head1 QUIC AND TLS
+
+QUIC reuses parts of the TLS protocol in its implementation. Specifically the
+TLS handshake also exists in QUIC. The TLS handshake messages are wrapped up in
+QUIC protocol messages in order to send them to the peer. Once the TLS handshake
+is complete all application data is sent entirely using QUIC protocol messages
+without using TLS - although some TLS handshake messages may still be sent in
+some circumstances.
+
+This relationship between QUIC and TLS means that many of the API functions in
+OpenSSL that apply to TLS connections also apply to QUIC connections and
+applications can use them in exactly the same way. Some functions do not apply
+to QUIC at all, and others have altered semantics. You should refer to the
+documenation pages for each function for information on how it applies to QUIC.
+Typically if QUIC is not mentioned in the manual pages then the functions apply
+to both TLS and QUIC.
+
+=head1 QUIC STREAMS
+
+QUIC introduces the concept of "streams". A stream provides a reliable
+mechanism for sending and receiving application data between the endpoints. The
+bytes transmitted are guaranteed to be received in the same order they were sent
+in without any loss of data or reordering of the bytes. A TLS application
```

Comment:
Maybe `sent out` instead of `sent in`?

Reply:
Hmmm..."sent in" seems more natural to me.

Reply:
Just drop the _in_.  It's clear enough just saying _sent_.

Reply:
@mattcaswell please do what @paulidale suggests